p5js/Crepe_Piano/index.js

```diff
@@ -0,0 +1,116 @@
+//CREPE VARIABLES
+let crepe;
+const voiceLow = 100;
+const voiceHigh = 500;
+let audioStream;
+let fft;
+
+//KEYBOARD VARIABLES
+const cornerCoords = [10, 40];
+const rectWidth = 90;
+const rectHeight = 300;
+const keyRatio = 0.58;
+const scale = ['C', 'C#', 'D', 'D#', 'E', 'F', 'F#', 'G', 'G#', 'A', 'A#', 'B'];
+
+//TEXT VARIABLES
+let currentNote = '';
+let currentText = '';
+let currentFreq = '';
+const textCoordinates = [(rectWidth*8) + cornerCoords[0], 150]
+
+
+function createCrepe() {
+  crepe = new ml5.Crepe(getAudioContext(), audioStream.stream);
+  console.log(crepe);
+  loop();
+}
+
+function getNoteFromMidiNum(midiNum) {
+  let note = scale[midiNum % 12];
+  return note;
+}
+
+function setup() {
+  createCanvas(1200, 400);
+  noLoop();
+  audioStream = new p5.AudioIn(function(err) {
+      console.error(err);
```

Comment:
totally minor but just some inconsistent indentation (4 spaces here instead of 2?), got to add that linter!